templates/zerver/help/import-data-from-slack.md

```diff
@@ -25,3 +25,21 @@ If the users are not from the same organization, you should change your organiza
 {!admin.md!}
 
 2. Disable the **Restrict new users to the following email domains** option.
+
+## Elements that are not mapped one to one
+
+- Message edit history. We only keep the latest revision of an edited message.
+- Simultaneous bold and italic formatting of a word. This is not yet supported
```

Comment:
What does our bugdown do, in this case?

Reply:
The frontend parser does the simultaneous bold+italic of ***bolditalic***, but the backend parser then converts it to *bolditalic*.